pyproject.toml

```diff
@@ -31,58 +31,65 @@ classifiers = ["Development Status :: 5 - Production/Stable",
                ]
 
 dependencies = [
-   'numpy>=1.10',
+   'numpy',
    'packaging',
    'h5py',
-   'fabio>0.9',
+   'fabio',
 ]
 
 [build-system]
 build-backend = 'mesonpy'
 requires = [
-    'meson-python>=0.11',
-    "meson>=1.1",
-    'ninja',
-    'wheel',
-    'Cython>=0.29.31',
-    'pyproject-metadata>=0.5.0',
-    "tomli>=1.0.0; python_version < '3.12'",
+    'meson-python',
+    'Cython',
```

Comment:
This looks to be enough, moved some of the extra ones to `dev` optional dependencies